releasenotes/notes/add-ecr-sx-equivalences-5b6fe73ec599d1a4.yaml

```
@@ -0,0 +1,11 @@
+---
+feature:
```

```suggestion
features:
```

sorry - I forgot that the sections are plurals when I was typing out the example